README.md

```diff
@@ -1,5 +1,8 @@
 # Keda Manager
 
+## Status
```

Comment:
Is this a real-time status check of the module? That would be amazing! Can all modules use that functionalitiy? 
Or just an example how that would look like?

Reply:
I've never tested it, but it should be updated automatically.
Yes, all modules can use it.